README.md

```diff
@@ -76,6 +76,7 @@ If you maintain a minikube package, please feel free to add it here.
     * [xhyve driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#xhyve-driver), [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [VMware Fusion](https://www.vmware.com/products/fusion)
 * Linux
     * [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [KVM](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#kvm-driver)
+    * NOTE: minikube also supports a '--vm-driver=none' option that runs the kubernetes components on the host and not in a VM.  docker is required to use this driver but no hypervisor.
```

Comment:
nit: Maybe follow the format of the other drivers and link to a new page https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#none-driver

There, you could list all the packages you need to be installed on your host for the none driver to work.  I think the exposition should probably be off the main readme.